networkx/algorithms/centrality/tests/test_load_centrality.py

```diff
@@ -34,6 +35,11 @@ def setup_class(cls):
         cls.D = nx.cycle_graph(3, create_using=nx.DiGraph())
         cls.D.add_edges_from([(3, 0), (4, 3)])
 
+    def test_normalized_load_centrality(self):
+        b=nx.load_centrality(self.G,normalized=True)
+        for i in b.values():
+            assert i >= 0 and i <=1
```

Comment:
So if you look at line 63-65 at https://app.codecov.io/gh/networkx/networkx/blob/main/networkx/algorithms/centrality/load.py the red block on line 65 signifies that the tests never go into this branch of code. That means we need to create a test case where `normalized` is True and the order of graph is less than 2. Order just means the number of nodes in the graph.

So an example like
``` python
G = nx.path_graph(2) # A graph with 2 nodes and one connection from node 0 to node 1
nx.load_centrality(G, normalized=True) # this should give {0: 0.0, 1: 0.0}
```

You need to translate this bit to something like

``` python
    def test_P2_load(self):
        G = self.P2
        c = nx.load_centrality(G)
        d = {0: 0.000, 1: 0.000}
        for n in sorted(G):
            assert c[n] == pytest.approx(d[n], abs=1e-3)
```

Also it really helps when you link the actual issue being solved, so having something like "fixes https://github.com/networkx/networkx/issues/6036" somewhere in the description helps in getting context about whats happening pretty quickly :)

Let me know if you have any questions.

Reply:
Thank you. Can you check the code and let me know if I should remove the function I added?